assembly/fabric8-ide-dashboard-war/src/app/workspaces/share-workspace/share-workspace.html

```diff
@@ -0,0 +1,17 @@
+<!--
+
+    Copyright (c) 2016-2018 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<div class="no-permissions-error" layout="row" layout-align="center center">
+  <span class="material-design icon-ic_error_outline_24px"></span>
```

Comment:
@amisevsk workspace sharing should still be enabled but with some limitations - https://github.com/redhat-developer/rh-che/issues/798. So, in this PR I would propose just to disable organizations

Reply:
Good point. Done.